.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,34 @@
+# Introduction 
+
+This outlines how to propose a change to admiral . For more detailed info about contributing to admiral, and other pharmaverse packages, please see the [development process guide](https://pharmaverse.github.io/admiral/articles/development_process.html) as well as other Developer Guides in the Articles section of [admiral website](https://pharmaverse.github.io/admiral/index.html)
+
+The goal is to enable users of `{admiral}` to contribute and test the contribution model for `{admiral}`. Adjustments might be made over time.
+ 
+Our contribution model is still in a testing phase so we advise you to contact us when an issue is created via [Slack](https://app.slack.com/client/T028PB489D3/C02M8KN8269) _(If you don't have access, use this [link](https://join.slack.com/t/pharmaverse/shared_invite/zt-yv5atkr4-Np2ytJ6W_QKz_4Olo7Jo9A) to join)_.  We can discuss details or align expectations if you are not familiar with the `{admiral}` philosophy and programming strategy. The team will try to review the issues within the next backlog meeting and give some initial feedback. Since we are not a 100% fully resourced software development team it might be that some issues will take longer to respond to depending on the amount of overall issues. 
+
+# Basics
+ * For each new contribution, the user creates an issue on the issue tab on [GitHub](https://github.com/pharmaverse/admiral/issues) to put it in our backlog. 
+ 
+ * Familiarize yourself with our [programming strategy](programming_strategy.html), guidance for [GitHub usage](git_usage.html) and [unit testing](unit_test_guidance.html).
+
+ * All [created issues](https://github.com/pharmaverse/admiral/issues) will be reviewed within the next backlog meeting and the creator will receive an initial feedback via a comment. Someone from the core development team will then triage new issues by assigning the appropriate labels (such as “user request” so we can easily identify new requests).
+ 
+ * Issues are meant to be taken on by users from the Pharma programming community and not exclusively by the `{admiral}` team from GSK and Roche.
+
+# Contribution Model
```

Comment:
can we still show the image as we do in the current vignette https://pharmaverse.github.io/admiral/articles/contribution_model.html#contribution-model?

Reply:
not sure @dinakar29?  it is a bit fuzzy image btw

Reply:
@thomas-neitmann what do you think? happy to leave out if you think the words alone are enough to explain

Reply:
Yeah image should be fine. Moving things around _could_ cause relative references to break, so that's definitely something to consider.